Proton/Sources/Swift/Editor/EditorView.swift

```diff
@@ -260,11 +300,20 @@ open class EditorView: UIView {
         set { richTextView.isScrollEnabled = newValue }
     }
     
+    public var showsVerticalScrollIndicator: Bool {
+        get { richTextView.showsVerticalScrollIndicator }
+        set { richTextView.showsVerticalScrollIndicator = newValue }
+    }
+    
     /// Gets or sets the insets for the text container's layout area within the editor's content area
     public var textContainerInset: UIEdgeInsets {
         get { richTextView.textContainerInset }
         set { richTextView.textContainerInset = newValue }
     }
+    
+    public func updatePlaceholderVisibility() {
+        richTextView.updatePlaceholderVisibility()
```

Comment:
This is intentionally kept internal and if there are issues/limitations with this, it should be handled differently. Consumers of Proton need now know about nor worry about how these things are taken care of internally.